### PR TITLE
Deployment docs

### PR DESCRIPTION
**Pull request by angrybayblade**
## Proposed changes

This PR updates docs to reflect latest API changes 

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

### CODE REVIEW

**Review by DavidMinarsch — CHANGES_REQUESTED** [review]

File: docs/quick_start.md [hunk 1]
```diff
@@ -99,22 +96,30 @@ In this case, we consider the [Hello World agent service](./hello_world_agent_se
         ]
         ```
 
-2. Use the CLI to download and build the images to deploy the [Hello World agent service](./hello_world_agent_service.md):
+
+3. Run a local deployment for the [Hello World agent service](./hello_world_agent_service.md):
```
Comment (jmor7690) [inline docs/quick_start.md:100]: Please, split this step into 2 steps and follow the convention used in the rest of the docs:
first, detail exactly what the command is going to do.
second, present the command.

Also it's confusing that this step is called "run a local deployment", and in step 4 we have "run the deployment".

**Review by jmor7690 — CHANGES_REQUESTED** [review]
Changes are requested. Also, the files under /demos need to be in sync, as they follow the same structure.

File: docs/autonomy.md [hunk 2]
```diff
@@ -24,21 +24,24 @@ Usage: autonomy deploy build [OPTIONS] [KEYS_FILE]
...
+                                  Logging level for runtime.
+  --packages-dir PATH             Path to packages dir (Use with dev mode)
+  --open-aea-dir PATH             Path to open-aea repo (Use with dev mode)
+  --open-autonomy-dir PATH        Path to open-autonomy repo (Use with dev
+                                  mode)
+  --remote                        To use a remote registry.
+  --local                         To use a local registry.
+  -p                              Ask for password interactively
+  --password PASSWORD             Set password for key encryption/decryption
+  --help                          Show this message and exit.
```
Comment (dvilelaf) [inline docs/autonomy.md:44]: These options are subject to change overtime. Maybe open a ticket to add a test for them.

File: docs/quick_start.md [hunk 3]
```diff
@@ -73,7 +64,13 @@ autonomy init --remote
 Follow the steps indicated below to download a demonstration agent service from the Service Registry, and deploy it locally using Docker Compose.
 In this case, we consider the [Hello World agent service](./hello_world_agent_service.md).
 
-1. Prepare a JSON file `keys.json` containing the addresses and keys of the four agents that make up the [Hello World agent service](./hello_world_agent_service.md). Below you have some sample keys for testing:
+1. Use the CLI to download the `valory/hello build` service. 
+    ```bash
+    autonomy fetch valory/hello_world:0.1.0:bafybeifp6bwabnksgsjnqiv35qnbxtxsegro55ln6ogcrs3hhcqxkezpf4 --remote --service
```
Comment (dvilelaf) [inline docs/quick_start.md:69]: Have you checked that the quickstart now works with the new changes?

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**